src/App.js

```diff
@@ -1,16 +1,93 @@
 import React from 'react';
+import TodoList from './components/TodoComponents/TodoList';
+import TodoForm from './components/TodoComponents/TodoForm';
+import './appStyles.css';
+
+const stuffToDo = [];
+
+
 
 class App extends React.Component {
-  // you will need a place to store your state in this component.
-  // design `App` to be the parent component of your application.
-  // this component is going to take care of state, and any change handlers you need to work with your state
-  render() {
-    return (
-      <div>
-        <h2>Welcome to your Todo App!</h2>
-      </div>
-    );
-  }
+    constructor() {
+      super();
+      this.state = {
+        allStuffToDo: stuffToDo,
+        inputText: '',
+      };
+    }
+    
+    handleChange = event => {
+      this.setState({
+        [event.target.name]: event.target.value
+      });
+      
+      // console.log(event.target.name);
+    };
+    
+    addTask = event => {
```

Comment:
conventionally methods like these begin with the word handle, this is purely opinion i just thought you might want to be aware of that